core/src/main/java/org/infinispan/persistence/sifs/NonBlockingSoftIndexFileStore.java

```diff
@@ -631,7 +684,7 @@ private ByteBuffer readAndCheckKey(FileProvider.Handle handle, EntryHeader heade
       return serializedKey;
    }
 
-   private ByteBuffer toBuffer(byte[] array) {
+   ByteBuffer toBuffer(byte[] array) {
```

Comment:
Change doesn't appear to be needed.

Reply:
Not anymore, yeah. I was toying with how to do this for the notifications. Let me revert.